doppler/assets/logs/doppler.yaml

```diff
@@ -0,0 +1,56 @@
+id: doppler
+metric_id: doppler
+backend_only: false
+facets:
+  - groups:
+      - User
+    name: User Name
+    path: usr.name
+    source: log
+  - groups:
+      - User
+    name: User Email
+    path: usr.email
+    source: log
+  - groups:
+      - Event
+    name: Type
+    path: type
+    source: log
+pipeline:
+  type: pipeline
+  name: Doppler
+  enabled: true
+  filter:
+    query: "source:doppler"
+  processors:
+    - type: date-remapper
+      name: Define `createdAt` as the official date of the log
+      enabled: true
+      sources:
+        - createdAt
+    - type: message-remapper
+      name: Define `text` as the official message of the log
+      enabled: true
+      sources:
+        - text
+    - type: attribute-remapper
+      name: Map `user.name` to `usr.name`
```

Comment:
Actually I am not very clear why you need to have these 2 remappers? You are already submitting logs with these property set:
```
                    "usr": {
      				"name": "Nic Manoogian",
      				"email": "nic.manoogian@doppler.com"
      			},
      			"link": "https://dashboard.doppler.com/workplace/30bd9e7d881446ce1bf8/logs?id=wpmULcSfO9sNxKdVUeaO0HK2",
      			"type": "services.datadog.connect",
      			"title": "Activity Log: wpmULcSfO9sNxKdVUeaO0HK2",
      			"user": {
      				"name": "Nic Manoogian",
      				"profile_image_url": "https://www.gravatar.com/avatar/7f447a6cfe4f594f91c149b9c9ca872d?s=500&d=retro",
      				"email": "nic.manoogian@doppler.com",
      				"username": "nic"
      			},
  ```

Our system will already keep these values without the remapper. Is there some logs that will not include `usr` data?

Reply:
Hi @thibaultkrebs, that `usr` field is not include in the payload that we submit to Datadog, it's only present in the log because of the remapper

Reply:
Hi @nmanoogian The sample should be a raw log before it is sent to Datadog. It sounds like you are saying that this sample is from Datadog after the logs went through the log pipeline. If that is the case, please update the sample to be a raw log sample before it is sent to Datadog.

Reply:
@nmanoogian bumping the above - please update the sample. Once done, logs can review again.